src/Appwrite/Auth/OAuth2/Etsy.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace Appwrite\Auth\OAuth2;
+
+use Appwrite\Auth\OAuth2;
+
+class Etsy extends OAuth2
+{
+    /**
+     * @var string
+     */
+    private string $endpoint = 'https://api.etsy.com/v3/public';
+
+    /**
+     * @var string
+     */
+    private $version = '2022-07-14';
```

Comment:
```suggestion
    private string $version = '2022-07-14';
```
As @TorstenDittmann  suggested, can you please add typings to all the variables ?